### ISSUE TITLE
Add option to configure path for caching file

### ISSUE THREAD

**Issue by naustica** [bganglia/unpaywall-python#35]

**Comment by bganglia** [comment]
@naustica So is the idea that the cache would take one argument for the filename and another for the directory path?

**Comment by naustica** [comment]
@bganglia Yes, I think that would be a nice feature. We could use os.getcwd() as the default path.
